src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -16,6 +16,10 @@ export interface StatementGroupModel {
     description?: string;
     conditionId?: string;
     conditionDefinition?: string;
+    createdAt: string;
```

Comment:
Isn't this supposed to be a date? (aka type `number` instead of `string`)

Reply:
will be fixed later